src/components/layout/TwoColumnsLayout.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { FC, ReactNode } from 'react';
+
+import { GridItem, Heading } from '@chakra-ui/react';
+
+import BaseGridLayout, { repeatArea } from './BaseGrid';
+import Link from '../link';
+import { ArrowLeftIcon } from '../icons';
+import Divider from '../divider';
+
+interface Props {
+  header: ReactNode;
+  title?: string;
+  backLink?: {
+    text: string;
+    url: string;
+  };
+  leftContent?: ReactNode;
+  rightContent?: ReactNode;
+}
+
+const TwoColumnsLayout: FC<Props> = ({
+  header,
+  title,
+  backLink,
+  leftContent,
+  rightContent,
+}) => {
+  return (
+    <BaseGridLayout
+      templateAreas={{
+        xs: `"header" "backlink" "heading" "rightContent" "leftContent"`,
+        lg: `
+            "${repeatArea(12, 'header')}"
+            "${repeatArea(12, 'backlink')}"
+            "${repeatArea(6, 'heading')} . ${repeatArea(5, 'rightContent')}"
+            "${repeatArea(6, 'leftContent')} . ${repeatArea(
+          5,
+          'rightContent'
+        )}"`,
+      }}
+      gridTemplateRows="minmax(min-content, max-content) minmax(min-content, max-content) minmax(min-content, max-content) 1fr"
+    >
+      <GridItem area="header">
+        {header}
+        <Divider />
```

Comment:
I will doble check with the designer if we can keep the divider inside the layout:

![Screenshot 2022-08-18 at 13 01 29](https://user-images.githubusercontent.com/37380787/185379763-e6b88a09-443e-449f-8f49-34ae440ddeb8.png)